.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+language: go
+
+branches:
+        only:
+                - merkle-tree
+
+notifications:
+  # Email notifications are disabled to not annoy anybody.
+  # See http://about.travis-ci.org/docs/user/build-configuration/ to learn more
+  # about configuring notification recipients and more.
+  email: false
+
+    Status API Training Shop Blog About 
```

Comment:
That's probably a copy-paste error. It didn't affect the Travis build, though...